JoinColony/colonyCDapp
# Finalizing expenditure as non-owner (using motions)

## Description

This PR introduces a new saga - `finalizeExpenditureMotion` - which allows users to create a motion to finalize an expenditure by calling `finalizeExpenditureViaArbitration` contract method.

## Testing

- Enable voting reputation in your colony
- Create an expenditure, lock and fund it, then use the "Finalize via motion" button
- Progress the motion to finalization
- Run the following query to verify expenditure status has changed and the motion is linked to the expenditure:
```gql
query ListExpenditures {
  listExpenditures(filter: {nativeId: {ge: 33}}) {
    items {
      nativeId
      status
      motions {
        items {
          id
          action {
            type
          }
        }
      }
    }
  }
}
```

Resolves #1978
